packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -20,23 +20,20 @@ import lens from "hubble";
 import * as React from "react";
 import _ from "underscore";
 
-import SortableArea from "../components/sortable";
-
 import type {
     PerseusExpressionWidgetOptions,
     LegacyButtonSets,
     ExpressionDefaultWidgetOptions,
+    PerseusExpressionAnswerForm,
 } from "@khanacademy/perseus-core";
 
-type ChangeFn = typeof Changeable.change;
-
 const {InfoTip} = components;
 
 type Props = {
-    widgetId?: any;
+    widgetId?: string;
     value?: string;
-} & Omit<PerseusExpressionWidgetOptions, "buttonsVisible"> &
-    Changeable.ChangeableProps;
+    onChange: (any) => void;
```

Comment:
Is there any way we can type this?  I'd expect that it would be something like this: 

```typescript
    onChange: (newValues: Partial<PerseusExpressionWidgetOptions>) => void;
```

This would mean we can't support the form of passing a string and new value, but I think with literal object syntax it's almost exactly the same "size"

ie. `this.props.onChange("buttonSets", buttonSets)` is actually more than `this.props.onChange({buttonSets})`